modules/openapi-generator/src/main/resources/python/python-experimental/model_test.mustache

```diff
@@ -11,22 +11,22 @@ import {{packageName}}
 {{#imports}}
 {{{.}}}
 {{/imports}}
-from {{modelPackage}}.{{classFilename}} import {{unescapedDescription}}
+from {{modelPackage}}.{{classFilename}} import {{classname}}
 
 
-class Test{{unescapedDescription}}(unittest.TestCase):
-    """{{unescapedDescription}} unit test stubs"""
+class Test{{classname}}(unittest.TestCase):
+    """{{classname}} unit test stubs"""
 
     def setUp(self):
         pass
 
     def tearDown(self):
         pass
 
-    def test{{unescapedDescription}}(self):
-        """Test {{unescapedDescription}}"""
+    def test{{classname}}(self):
+        """Test {{classname}}"""
         # FIXME: construct object with mandatory attributes with example values
-        # model = {{unescapedDescription}}()  # noqa: E501
+        # model = {{classname}}()  # noqa: E501
```

Comment:
Thank you for this fix! I missed this when I switched python-experimental to using classname.